core/src/test/java/kafka/testkit/KafkaClusterTestKit.java

```diff
@@ -238,7 +238,7 @@ public KafkaClusterTestKit build() throws Exception {
                                 bootstrapMetadata);
                     } catch (Throwable e) {
                         log.error("Error creating controller {}", node.id(), e);
-                        Utils.swallow(log, "sharedServer.stopForController", () -> sharedServer.stopForController());
+                        Utils.swallow(log, "sharedServer.stopForController", () -> sharedServer.stopForController(), null);
```

Comment:
nit: Should we add an overload to avoid having to pass null here?

Reply:
ah! intended to do that. Have done it now.